aws.tf

```diff
@@ -25,11 +25,11 @@ resource "aws_route53_record" "service" {
 }
 
 resource "aws_route53_record" "service_validation" {
-  name            = fastly_tls_subscription.service.managed_dns_challenge.record_name
-  type            = fastly_tls_subscription.service.managed_dns_challenge.record_type
+  name            = fastly_tls_subscription.service.managed_dns_challenges.record_name
+  type            = fastly_tls_subscription.service.managed_dns_challengs.record_type
```

Comment:
```suggestion
  type            = fastly_tls_subscription.service.managed_dns_challenges.record_type
```
?

Reply:
Opps!! Typo error, I will update it. Thanks for check it.